File: src/codes/Users_Manual/silencer/silencer_fixed_completion_steps.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~int main() {
//~using namespace autd3;
Silencer{FixedCompletionTime{.intensity = std::chrono::microseconds(250),
                             .phase = std::chrono::microseconds(250),
                             .strict = true}};
//~return 0; }

File: src/codes/Users_Manual/silencer/silencer_fixed_completion_steps.rs
# use autd3::prelude::*;
# use std::time::Duration;
# fn main() {
# let _ = 
Silencer {
    config: FixedCompletionTime {
        intensity: Duration::from_micros(250),
        phase: Duration::from_micros(250),
        strict: true,
    },
};
# }